src/libcollections/vec.rs

```diff
@@ -2162,6 +2171,15 @@ pub struct PlaceBack<'a, T: 'a> {
     vec: &'a mut Vec<T>,
 }
 
+#[stable(feature = "collection_debug", since = "1.15.0")]
+impl<'a, T: 'a + fmt::Debug> fmt::Debug for PlaceBack<'a, T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_tuple("Vec::PlaceBack")
+         .field(&self.vec.as_slice())
+         .finish()
+    }
+}
```

Comment:
The habit, didn't thought of testing it, my bad... However, once this PR merged, such cases won't happen again thanks to the corresponding deny option. ;)